src/priceHistory/saga.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Price,
+  fetchPriceHistoryFailure,
+  fetchPriceHistoryStart,
+  fetchPriceHistorySuccess,
+} from 'src/priceHistory/slice'
+import Logger from 'src/utils/Logger'
+import { ensureError } from 'src/utils/ensureError'
+import { fetchWithTimeout } from 'src/utils/fetchWithTimeout'
+import { safely } from 'src/utils/safely'
+import networkConfig from 'src/web3/networkConfig'
+import { call, put, spawn, takeLatest } from 'typed-redux-saga'
+
+const TAG = 'priceHistory/saga'
+
+/** @beta - Exclude from Knip dep check */
```

Comment:
this is great! seems perfect for our use case

Reply:
but is it actually needed? seems like this function is used by the saga below it

Reply:
It got flagged here for an unused export: https://github.com/valora-inc/wallet/actions/runs/7306296211/job/19910864603.

Reply:
is it still flagged now that there is a test?

Reply:
in general, if an export is unused and you don't actually plan to use it outside the file, seems like you should just not export it, instead of adding a beta tag to get knip to ignore it. seems like we should use beta tags for beta features, like things that we plan to use but want to build out in a separate PR

Reply:
The function is no longer flagged by knip after being used in a test.